static/_redirects

```diff
@@ -179,6 +179,9 @@ support/how-to/rackspace-email-password-recovery-faq/  /support/how-to/manage-pa
 
 # Specific 404 Redirects
 
+/support/how-to/aiops-alert-monitoring/ /support/how-to/intro-to-aiops-and-rns-configuration/404
+/support/how-to/intro-to-aiops-and-rns-configuration/ /support/how-to/intro-to-aiops-and-rns-configuration/ 301
```

Comment:
/how-to/intro-to-aiops-and-rns-configuration/     /support/how-to/intro-to-aiops-and-rns-configuration/    301